src/lib/core/questions.ts

```diff
@@ -549,6 +549,31 @@ export const questionGroups: QuestionGroup[] = [
 						? "TypeScript"
 						: "JavaScript",
 			},
+			{
+				condition: [{ name: "features", contains: "adapter" }],
+				type: "select",
+				name: "nodeVersion",
+				label: "Node.js version",
+				expert: true,
+				optional: true,
+				message:
+					"What's the minimum Node.js version you want to support?",
+				initial: "12",
```

Comment:
Do we really want to make version 12 the default? It seems to already have gone out of "maintenance" mode by end of April:
https://nodejs.org/en/about/releases/

Reply:
I would even remove it completely

Reply:
If an adapter can run on 12, why should we not allow it? Unless someone uses the new JS features, this shouldn't make a difference.

We could add a hint to 12 that it is EOL and make 14 the default.